usr/share/rear/backup/NETFS/default/500_make_backup.sh

```diff
@@ -178,9 +224,9 @@ sleep 1
 case "$(basename $BACKUP_PROG)" in
     (tar)
         if (( $backup_prog_rc == 1 )); then
-            LogPrint "WARNING: $(basename $BACKUP_PROG) ended with return code $backup_prog_rc and below output:
+            LogPrint "WARNING: $(cat $TMP_DIR/failing_backup_prog) ended with return code $backup_prog_rc and below output:
```

Comment:
Above that code part there  is a comment that reads
<pre>
# everyone should see this warning, even if not verbose
</pre>
which shows that using `LogPrint` is just another bug here
because `LogPrint` calls `Print` and `Print` only output in verbose mode
cf. lib/_input-output-functions.sh
which is the reason why I introduced additional functions
like `LogUserOutput` and `LogPrintError` but of couse that
did not magically fix all places where `LogPrint` was falsely used.

Reply:
ok